src/TLuaInterpreter.cpp

```diff
@@ -8413,7 +8413,6 @@ int TLuaInterpreter::setScript(lua_State* L)
     auto [id, message] = pLuaInterpreter->setScriptCode(name, luaCode, pos);
     lua_pushnumber(L, id);
     if (id == -1) {
-        lua_pushboolean(L, false);
```

Comment:
Yeah, this **was** wrong anyhow as in the error situation you'd be pushing three items on to the stack but telling Lua that there was only 2 it needed to handle...